### PR TITLE
Update powershell 6.0.0-rc Add GSSAPI support to caveat

### PR DESCRIPTION
**Pull request by markekraus**
After making all changes to the cask:

- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` reports no offenses.
- [X] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [ ] Named the cask according to the [token reference].
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked there are no [open pull requests] for the same cask.
- [ ] Checked the cask was not already refused in [closed issues].
- [ ] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

ref PowerShell/PowerShell#5629

### CODE REVIEW

**Review by commitay — CHANGES_REQUESTED** [review]
Can you update the reference in the caveat for the additional flag please?

If it's covered by upstream docs linking to them would be preferred.

File: Casks/powershell.rb [hunk 1]
```diff
@@ -28,7 +28,8 @@
              ]
 
   caveats <<~EOS
-    A OpenSSL-backed libcurl is required for custom handling of certificates.
+    A OpenSSL-backed libcurl with GSSAPI is required for custom handling of certificates
+    and default credentials for web requests.
     This is rarely needed, but you can install it with
       brew install curl --with-openssl --with-gssapi
     See https://github.com/PowerShell/PowerShell/issues/2211
```
Comment (commitay) [inline Casks/powershell.rb]: This link needs to be updated as well to cover both flags

Comment (markekraus) [reply Casks/powershell.rb]: swapped it for a new link

Comment (commitay) [reply Casks/powershell.rb]: Thanks!

**Review by commitay — APPROVED** [review]

**Comment by commitay** [comment]
Thanks for the PR!

**Pull request merged**